source/extensions/filters/udp/dns_filter/dns_filter_resolver.h

```diff
@@ -18,12 +18,12 @@ enum class DnsFilterResolverStatus { Pending, Complete, TimedOut };
  */
 class DnsFilterResolver : Logger::Loggable<Logger::Id::filter> {
 public:
-  DnsFilterResolver(DnsFilterResolverCallback& callback, AddressConstPtrVec resolvers,
+  DnsFilterResolver(DnsFilterResolverCallback& callback, AddressConstPtrVec /*resolvers*/,
                     std::chrono::milliseconds timeout, Event::Dispatcher& dispatcher,
                     uint64_t max_pending_lookups,
-                    const envoy::config::core::v3::DnsResolverOptions& dns_resolver_options)
+                    const envoy::config::core::v3::DnsResolverOptions& /*dns_resolver_options*/)
       : timeout_(timeout), dispatcher_(dispatcher),
-        resolver_(dispatcher.createDnsResolver(resolvers, dns_resolver_options)),
+        //resolver_(dispatcher.createDnsResolver({}, {})),
```

Comment:
Is the `resolver_` field in this class not being used?

Reply:
It most certainly is used.  I am guessing that this is temporary since this PR is WIP.

Reply:
Yes, this is temporary. I am working on the actual implementation for resolver_ field.  The plan is to passing the proto configuration down to the dispatcher function, same as other cases.

Reply:
ACK, the latest code has the change here.